browser/profiles/brave_profile_manager.cc

```diff
@@ -117,6 +118,9 @@ void BraveProfileManager::DoFinalInitForServices(Profile* profile,
     return;
   brave_ads::AdsServiceFactory::GetForProfile(profile);
   brave_rewards::RewardsServiceFactory::GetForProfile(profile);
+  brave::BraveFederatedLearningService* federated_learning_service =
+      brave::BraveFederatedLearningServiceFactory::GetForProfile(profile);
+  federated_learning_service->Start();
```

Comment:
what is the reason for having a separate start method? Also if this can't be lazy loaded the preferred method is returning true from `ServiceIsCreatedWithContext` in the factory

Reply:
Included Start logic inside the constructor and added the preferred method in the factory 👍